### DISCUSSION TITLE
windows 不能运行

### DISCUSSION THREAD

**Discussion by MarshalT** [illacloud/illa-builder#753] [Q&A]
PS C:\Users\prm\Desktop> illa deploy --self --port=10000 --server-addr=<127.0.0.1>
 Running a self-hosted installation...
 No running docker found.
 Please check the status of docker.

**Comment by karminski** [comment]
@MarshalT  hi, 运行环境需要docker, 请安装docker: https://www.docker.com/products/docker-desktop/
